core-bundle/src/EventListener/PreviewAuthenticationListener.php

```diff
@@ -70,6 +70,8 @@ public function __invoke(RequestEvent $event): void
             UrlGeneratorInterface::ABSOLUTE_URL
         );
 
+        $url = str_replace($request->getScriptName(), '', $url);
```

Comment:
Is a string replace a good idea here? Shouldn’t we instead change the router context by calling `$this->router->getContext()` and `$this->router->setContext()`?

Reply:
I thought about the same, but was not sure if it's a good idea? 🤔 I would actually prefer it.

Reply:
Adjusted in b23ccafda.